ui/src/controls/ToggleDarkModeButton.svelte

```diff
@@ -0,0 +1,32 @@
+<svelte:head>
+{#if stored == "dark"}
+<link rel='stylesheet' href='darkmode.css' id="dark-mode-css">
+{/if}
+</svelte:head>
+
+<script>
+import { createEventDispatcher } from 'svelte';
+const dispatch = createEventDispatcher();
+var stored = localStorage.getItem("dark-mode-storage");
+
+if (!stored) {
+  if (window.matchMedia("(prefers-color-scheme: dark)").matches ) {
+    stored = "dark";
+  } else {
+    stored = "light";
+  }
+}
+
+function switchTheme() {
+    if (stored == "light") {
+        stored = "dark";
+    } else {
+        stored = "light";
+      }
+}
+</script>
+
+<button on:click="{switchTheme}">
+<img src="img/icon_darkmode.svg" alt="dark mode toggle" class="no-responsive no-border"/>
```

Comment:
I'm not completely sure if we should wrap the icon in a button, or if we should let it stand on its own. I'll experiment a bit with that myself and see how it looks in the navbar :)

Reply:
Maybe with a span tag as in https://github.com/liwde/blinddichten/pull/1/commits/4a501f63306436014bb80975c50d0590f93fcde8?

Reply:
I've worked out something

Reply:
I've committed it, let me know what you think :)

Reply:
Looks like you already fixed it in your commit https://github.com/liwde/blinddichten/pull/1/commits/f2eaaebf41574044c1e57a801a7777f0b4703740. I like the way it looks.